### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mtannerfors**
After hours of debugging `Error: 3 INVALID_ARGUMENT: Request contains an invalid argument.` turns out that apiEndpoint is required if the processor is based in the eu cluster, but not if using the US. Suggest this is added to the Readme.MD.

Thanks for otherwise amazing products and service.

### CODE REVIEW

**Review by bcoe — CHANGES_REQUESTED** [review]
@mtannerfors the sample in the README is automatically generated from here:

https://github.com/googleapis/nodejs-document-ai/blob/master/samples/quickstart.js

(so if you add a comment here, it will be reflected in the README).

Perhaps rather than the note specifically about the EU region, you could link to the list of regions?

https://cloud.google.com/ai-platform/training/docs/regions

**Comment by bcoe** [comment]
@mtannerfors is this still something you'd like to land? mind making the edit in `samples/quickstart.js`?

**Comment by mtannerfors** [comment]
> @mtannerfors is this still something you'd like to land? mind making the edit in `samples/quickstart.js`?

Sorry for laggy response, just updated samlples/quickstart.js a suggested.

File: README.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ const {
 } = require('@google-cloud/documentai').v1beta3;
 
 // Instantiates a client
+// const client = new DocumentProcessorServiceClient({apiEndpoint: 'eu-documentai.googleapis.com'}); // Required if using eu based processor
```
Comment (bcoe) [inline README.md:74]:
```suggestion
// You can use processors based in other regions, using the apiEndpoint option:
// const client = new DocumentProcessorServiceClient({apiEndpoint: 'eu-documentai.googleapis.com'}); 
```

**Comment by telpirion** [comment]
FYI The list of [regions listed for AI Platform](https://github.com/googleapis/python-documentai/pull/108) is not the same as what's available for Document AI.

The current list of regional support for Document AI is [here](https://cloud.google.com/document-ai/docs/regions).

**Comment by JustinBeckwith** [comment]
Looks like this will be covered in #180 :)

**Pull request closed without merging**